src/service/history-impl.js

```diff
@@ -143,15 +143,20 @@ export class History {
   }
 
   /**
-   * Requests navigation one step back. This request is only satisifed
-   * when the history has at least one step to go back in the context
-   * of this document.
+   * Requests navigation one step back. This first attempts to go back within
+   * the context of this document. If there is no history left, then attempt
+   * the window's history.back.
+   *
+   * @param {boolean=} canPerformWindowBack
    * @return {!Promise}
    */
-  goBack() {
+  goBack(canPerformWindowBack) {
```

Comment:
By "parameterize" we meant the external API e.g. `on="tap:AMP.goBack(window=true)"`. This would avoid changing user-visible behavior on existing pages. 

What we were discussing is whether or not this is actually needed. I'm happy with a 5-min chat in design review and publishing an I2S.